### DISCUSSION TITLE
Which minWriterVersion and minReaderVersion does Trino support for Delta Lake?

### DISCUSSION THREAD

**Discussion by LucaSoato** [trinodb/trino#21035] [Q&A] [answered]
Hi Trino Community,
I'm looking into how Trino works with Delta Lake, particularly around version compatibility for minWriterVersion and minReaderVersion. Could anyone clarify which minWriterVersion and minReaderVersion Trino currently supports for Delta Lake?
Thanks and have a wonderful day
Luca

**Comment by ebyhr** [comment, marked as answer]
Reader Version 3 and writer Version 7 are supported. Note that it doesn't mean the connector supports all features. For instance, writing deletion vectors isn't supported. See https://trino.io/docs/current/connector/delta-lake.html#delta-lake-table-features for more details.

**Reply by LucaSoato** [reply]
Thanks for the help!
